enhancements/cert-manager/istio-csr-controller.md

```diff
@@ -123,27 +123,27 @@ CR, which is enforced in the CRD using CEL validations.
 - `.spec.istioCSRConfig.istio.revisions`
 - `.spec.istioCSRConfig.istio.namespace`
 
-When an OpenShift user deletes `istiocsr` CR object, `istio-csr` controller will stop managing all the resources created
-for installing `istio-csr` agent and user will manually clean up the resources. Please refer `Operational Aspects of API Extensions`
-section for command to list all the resources.
+When an OpenShift user deletes `istiocsr.operator.openshift.io` CR object, `istio-csr-controller` will stop managing all
+the resources created for installing `istio-csr` agent and user will manually clean up the resources. Please refer
+`Operational Aspects of API Extensions` section for command to list all the resources.
 
-`istiocsr` CR status sub-resource will be used for updating the status of the `istio-csr` agent installation, any error
-encountered while creating the required resources or the reconciling the state.
+`istiocsr.operator.openshift.io` CR status sub-resource will be used for updating the status of the `istio-csr` agent
+installation, any error encountered while creating the required resources or the reconciling the state.
 
 A fork of [upstream](https://github.com/cert-manager/istio-csr) `istio-csr` will be created [downstream](https://github.com/openshift/cert-manager-istio-csr)
 for better version management.
 
 ### Workflow Description
 
-- Enable `istio-csr` controller
-  - An OpenShift user enables `istio-csr` controller by creating the `istiocsr` CR.
-  - `istio-csr` controller based on the configuration in `istiocsr` CR, deploys `istio-csr` agent in the CR created
-    namespace.
+- Enable `istio-csr-controller`
+  - An OpenShift user enables `istio-csr-controller` by creating the `istiocsr.operator.openshift.io` CR.
+  - `istio-csr-controller` based on the configuration in `istiocsr.operator.openshift.io` CR, deploys `istio-csr` agent
+    in the CR created namespace.
 
 ![alt text](./istio-csr-create.png).
 
-- Disable `istio-csr` controller
-  - An OpenShift user disables `istio-csr` controller by deleting the `istiocsr` CR.
+- Disable `istio-csr-controller`
+  - An OpenShift user disables `istio-csr-controller` by deleting the `istiocsr.operator.openshift.io` CR.
```

Comment:
User is not aware of the controller in the operator. Maybe this should read as , istio-csr-controller is stopped on deletion of the CR.